interfaces.go

```diff
@@ -40,7 +40,11 @@ type Growth interface {
 	GetFreeTrialInfo(context.Context, int32) ([]*GrowthPromoCodeClaim, error)
 }
 
-type LoginRealm interface {
+// User service allows managing users in Confluent Cloud
+type UserInterface interface {
+	List(context.Context) ([]*User, error)
+	GetServiceAccounts(context.Context) ([]*User, error)
+	GetServiceAccount(context.Context, int32) (*User, error)
 	LoginRealm(context.Context, *GetLoginRealmRequest) (*GetLoginRealmReply, error)
```

Comment:
It seems like we're combining two APIs into one here? Should the user management / login realm functions really be grouped together?

Reply:
They were together...  I renamed it to `loginRealm` as I thought no other `User` methods would be needed.